content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Menjalankan Tugas-Tugas Otomatis dengan CronJob
+min-kubernetes-server-version: v1.8
+content_template: templates/task
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kamu dapat menggunakan {{< glossary_tooltip text="CronJob" term_id="cronjob" >}} untuk menjalankan {{< glossary_tooltip text="Job" term_id="job" >}} yang dijadwalkan berbasis waktu. Job akan berjalan seperti pekerjaan-pekerjaan [Cron](https://en.wikipedia.org/wiki/Cron) di Linux atau sistem UNIX.
+
+CronJob sangat berguna untuk membuat pekerjaan yang berjalan secara periodik dan berulang, misalnya menjalankan pencadangan atau mengirim email.
```

Comment:
does pencadangan mean "backup"? if so let's add parentheses on more familiar word or original one to readers
```suggestion
CronJob sangat berguna untuk membuat pekerjaan yang berjalan secara periodik dan berulang, misalnya menjalankan pencadangan (_backup_) atau mengirim email.
```

Reply:
done